### ISSUE TITLE
Allow re-ordering log filters

### ISSUE THREAD

**Issue by gaetanquentin** [rundeck/rundeck#3807]
Question: how to set log filter order? If i have 3 log filter and want to add a 4th between them, how to do it? Must i delete all and recreate them in the targeted order?

**Comment by gschueler** [comment]
yes right now the order is set as you add them, so to change the order you would have to remove and re-add them in the right order. 

I will rename this issue as an enhancement request to allow reordering them.

**Comment by gaetanquentin** [comment]
and allow to deactivate them without deleting them.

**Comment by gaetanquentin** [comment]
feature still needed..

**Comment by fdevans** [comment]
Thanks for the report. As part of a cleanup of our issue tracker, we're closing issues older than 12 months, in line with Rundeck's support policy of one year from a release's date (see the [release calendar](https://docs.rundeck.com/docs/history/release-calendar.html) for currently supported versions). This isn't a judgment that the report is invalid — many of these are likely still real — but we can't verify against a version this old.

If you're still seeing this on a currently supported release (6.1.x or 5.20.x+), please open a new issue using our [bug report template](https://github.com/rundeck/rundeck/issues/new?template=bug_report.yml) with your version and reproduction steps. Please don't reopen this issue — a fresh issue keeps things tracked against a supported version and gets it in front of us properly. We're actively working on improving how issues get submitted and triaged going forward.

Thanks for your patience,
The Rundeck Team
